src/patternfly/components/Page/page.scss

```diff
@@ -9,30 +9,50 @@
   // Header brand
   --pf-c-page__header-brand--PaddingTop: var(--pf-global--spacer--sm);
```

Comment:
Looks like you can also remove this as its declared but not used.

Reply:
updated